index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Peach CSS - Component Library</title>
+</head>
+
+<body>
+    <nav class="navigations">
+        <h1 class="nav-head">PeachCSS</h1>
+        <input class="nav-input" placeholder="🔍 Quick Search Anything" />
+        <small>v0.0</small>
+    </nav>
+    <div class="divider"></div>
+    <section >
+        <h1 class="section-header">Build fast, responsive Components with PeachCSS !!</h1>
+        <p>PeachCSS is a collection of CSS styles to make simple websites look authentic, Using power of CSS flexbox and CSS grids to create beautiful & responsive designs ready to use for your project.</p>
+        <a class="sec-btn" href="/main.html">Documentation</a>
```

Comment:
Always try to maintain consistency, to style a button always use a common class and then use extra classes/utility class to change the default styles. Example -> <a class=""btn btn-primary"></a>

Reply:
sure i will change you mentioned